src/shared_realm.cpp

```diff
@@ -554,5 +555,110 @@ util::Optional<int> Realm::file_format_upgraded_from_version() const
     return util::none;
 }
 
+#define VERSION_CAST(version_id) {(version_id).version, (version_id).index}
```

Comment:
If I could define this conversion on `HandoverPackage`, it'd probably define implicit conversions on `VersionID`. I can't put anything about `SharedGroup::VersionID` in the header though, so I'm a bit out of luck.